packages/alfa-css/src/value/gradient.ts

```diff
@@ -0,0 +1,224 @@
+import { Equatable } from "@siteimprove/alfa-equatable";
+import { Iterable } from "@siteimprove/alfa-iterable";
+import { Serializable } from "@siteimprove/alfa-json";
+import { Option, None } from "@siteimprove/alfa-option";
+import { Parser } from "@siteimprove/alfa-parser";
+
+import * as json from "@siteimprove/alfa-json";
+
+import { Token } from "../syntax/token";
+
+import { Color } from "./color";
+import { Length } from "./length";
+import { Percentage } from "./percentage";
+
+const { either, pair, map, option, oneOrMore, delimited, left, right } = Parser;
+
+/**
+ * @see https://drafts.csswg.org/css-images/#gradients
+ */
+export type Gradient = Linear;
+
+export namespace Gradient {
+  export type JSON = Linear.JSON;
+
+  /**
+   * @see https://drafts.csswg.org/css-images/#color-stop
+   */
+  export class Stop<
```

Comment:
Should Stop and Hint be moved to separate files in a subfolder? Like it's done with colours, for example.

Reply:
The reason for them to live in the `Gradient` namespace is that they're not really useful on their own as they are only used within the definitions of the individual gradient kinds.